mon-pix/app/styles/components/_user-account.scss

```
@@ -61,7 +66,7 @@
 
       &__link {
         display: block;
-        padding: 16px 60px 16px 20px;
+        padding: 16px 16px 16px 20px;
```

```suggestion
        padding: var(--pix-spacing-4x);
``` 
Pour utiliser les tokens [spacing](https://ui.pix.fr/?path=/docs/utiliser-pix-ui-design-tokens-espacements--docs) du DS 😄